Sources/SmileID/Classes/Networking/Models/PartnerParams.swift

```diff
@@ -32,3 +32,38 @@ public struct PartnerParams: Codable {
         )
     }
 }
+
+extension PartnerParams {
```

Comment:
nit: why make this an extension and now just put it in the class since it's in the same file?